engine/render/renderer/Renderer3D.h

```diff
@@ -1,16 +0,0 @@
-//
-// Created by Aryeh Zinn on 29/9/2022.
-//
-
-#ifndef SIEGE_ENGINE_RENDERER3D_H
-#define SIEGE_ENGINE_RENDERER3D_H
-
-namespace Siege
```

Comment:
Looks like you accidentally checked these in again on your last PR

Reply:
Damn. Once we catch up all the branches we'll be able to get rid of this stupid thing.